Sources/ApolloCodegenLib/Frontend/JavaScript/package.json

```diff
@@ -13,7 +13,8 @@
     "npm": ">=7"
   },
   "dependencies": {
-    "graphql": "file:./vendor/graphql-16.0.0-pre+modern-js.tgz"
+    "graphql": "file:./vendor/graphql-16.0.0-pre+modern-js.tgz",
```

Comment:
Should this and the associated file be updated to the latest 16.0.0 release? Right now that would be `v16.0.0-rc.2`.

Reply:
@martijnwalraven probably knows

Reply:
I'm going to leave this as-is for now. I think we should revisit the question once graphql-js 16.0.0 is officially released.